deploy/Chart/templates/database/statefulset.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: database-sts
+  namespace: "{{ .Release.Namespace }}"
+  labels:
+    control-plane: database
+    app.kubernetes.io/name: database
+    app.kubernetes.io/part-of: radius
+spec:
+  serviceName: "database"
+  replicas: 1
```

Comment:
What if we have/need 3 instances of the same database? How would the data I/O be affected?

Reply:
This is something we will need to test

Reply:
This is something we will explicitly not support.

Reply:
To clarify for anyone looking at this, we're bundling Postgres for non-production use of Radius. This will be optional. 

The docs will recommend you manage **your own** database for production scenarios. This could be in Kubernetes, or in the cloud, or on premises, or whatever you choose.

-- 

It's a non-goal for us to become your production support team for Postgres.